CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Not released
 
+## 0.9.0
+
+*  Add support for Rails 7 and update Restforce dependency to newer version. Add `has_one` association. Model generator enhancements: automatically add types to fields, sort fields alphabetically, add `table_name` to class, and add optional namespace parameter so generated models can be namespaced. Add get/set via `[]` and `[]=` for `SObject` attributes. ([#3][])
```

Comment:
Let's split these sentences into separate bullets like you have in the PR description and we should be good to go.

Reply:
Roger. I figured since they all pointed back to the same PR it made more sense to do it this way.

Reply:
@spacechurro all done.